vendor/init.bat

```diff
@@ -165,3 +182,12 @@ if exist "%CMDER_ROOT%\config\user-profile.cmd" (
     echo.
     ) > "%CMDER_ROOT%\config\user-profile.cmd"
 )
+
+exit /b
+
+::
+:: sub-routines below here
+::
+:verbose-output
```

Comment:
I don't understand the comment.  Are you saying there should be a log_debug and a log_info subrouteen also?